product_docs/docs/biganimal/release/overview/03_security/compliance_and_certifications.mdx

```diff
@@ -0,0 +1,59 @@
+
+SOC 2 reports are unique to each organization. In line with specific business practices, each designs its own controls to comply with one or more of the trust principles.
+
+Type I classification describes a vendor’s systems and whether their design is suitable to meet relevant trust principles.
+
+SOC 2 Type I compliance for BigAnimal certifies that:
+
+- BigAnimal securely stores the customer data by providing end-to-end encryption. 
+
+- The customer application and data are protected behind the network or application firewall. Also, BigAnimal can successfully handle intrusion detection and any security incidents. 
+
+- BigAnimal supports access control and two-factor authentication for an added level of security and privacy. 
+
+- BigAnimal provides the capabilities for performance and process monitoring, including disaster recovery and quality assurance that ensures high availability.
+
+For more information, see [SOC 2 Compliance](https://www.imperva.com/learn/data-security/soc-2-compliance/) and [SOC 1 vs. SOC 2](https://linfordco.com/blog/soc-1-vs-soc-2-audit-reports/).
+
+## SOC 2 Type II
+
+Like SOC 2 Type I, a SOC 2 Type II report is also an internal controls report capturing how a company safeguards customer data and how well those controls are operating. Companies that use cloud service providers use SOC 2 reports to assess and address the risks associated with third party technology services. Independent third party auditors issue these reports that cover the principles of security, availability, confidentiality, and privacy.
+
+SOC 2 Type I is different from Type II because a Type I assesses the design of security processes at a specific point in time. While a Type II report assesses how effective those controls are over time by observing operations for six months.
+
+For more information, see [SOC 2 Type II](https://www.onelogin.com/compliance/soc-2-type-2) and [SOC 2 Type I vs. Type II](https://blog.rsisecurity.com/soc-2-type-1-vs-type-2-whats-the-difference/).
+
+## GDPR
+
+The General Data Protection Regulation (GDPR) is a regulation in European Union (EU) law on data protection and privacy, as well as in the European Economic Area (EEA). It also addresses the transfer of personal data outside the EU and EEA areas. The GDPR's primary aim is to enhance individuals' control and rights over their personal data and to simplify the regulatory environment for international business.
+
+GDPR compliance implies both privacy and security mechanisms definition, enforcement, and control, including evidence collection. 
+BigAnimal supports GDPR at service level, which means BigAnimal protects the personal data and privacy of EU citizens for transactions that occur in EU member states.
+
+For more information, see [GDPR Principles and Regulations](https://gdpr-info.eu/).
```

Comment:
I would remove this

Reply:
I can't tell exactly what content you're referring to with these comments: the "For more information" paragraph or the whole section?